src/local-history/local-history-manager.ts

```diff
@@ -23,12 +23,13 @@ export const LOCAL_HISTORY_DIRNAME = '.local-history';
 
 export class LocalHistoryManager {
 
-    private historyFilesForActiveEditor: HistoryFileProperties[] = [];
```

Comment:
I think we can keep it private and have an accessor for it.
I would not want other areas of the code mutating the variable.

Reply:
@vince-fugnitto I changed it to public because we used it here:
https://github.com/vince-fugnitto/local-history-ext/blob/65588b154077345d30a68c6b4c8ce614307eb243/src/local-history/local-history-tree-provider.ts#L39

Reply:
I will add a getter for this private variable.